src/app/content/highlights/components/Card.tsx

```diff
@@ -43,9 +44,12 @@ export interface CardProps {
   data?: HighlightData;
   className: string;
   zIndex: number;
+  // If this prop is true then focus will be moved to this card.
+  shouldFocusCard: boolean;
   topOffset?: number;
   highlightOffsets?: { top: number, bottom: number };
   onHeightChange: (ref: React.RefObject<HTMLElement>) => void;
+  moveFocusToTheHighlight: () => void;
```

Comment:
it is moving focus to the screenreader node before highlight

Reply:
I could modify `Highlight.focus()` method to do the same, but we still have to reset `shouldFocusCard` state because if we don't then focusing another highlight would automatically move focus to the card
It is behaving like this because it is implemented as a hook that triggers focus if this props is set.
I was thinking on giving unique data-attr for each card and focus it using normal js, but then we can have problems related to showing cards without notes (we can't focus them until they show up)

Reply:
`shouldFocusCard` makes some sense to me. i think `moveFocusToTheHighlight` would make more sense if we just called the regular focus function and had that focus the SR element.

for `shouldFocusCard`, you might want to consider putting it in redux state and resetting it to false on any highlight focus in/out actions and then only set it to true when the key combo is pressed. you might find that easier to manage than all the react-hooks stuff